lib/payment_provider/handlers/async_handler.rb

```diff
@@ -11,6 +11,7 @@ class AsyncHandler
       # 'customer.subscription.created' => PaymentProvider::Handlers::SubscriptionHandler,
 
       def call(event)
+        ::Rails::logger.error("WEBHOOK: #{event.type} CONNECT: #{event.try(:account)} LIVEMODE: #{event.livemode}")
```

Comment:
Reason it is error vs info (if there is such a level on Rails logger)?

Reply:
There is such a log level, I didn't want it filtered, but info would also work on our prod. Normally prod log level would be higher.